### DISCUSSION TITLE
How Can I Easily Refresh My Repo to Show A Changed LICENSE File?

### DISCUSSION THREAD

**Discussion by ChrisMarshallNY** [community/community#22058] [New to GitHub]
<p>This should be a simple question, but I haven’t found any answers. I may have missed something. If so, I would be grateful to be directed to the answer.</p>
<p>I had a LICENSE file in several of my repos that was not parsed by GH (fair enough. I had added the URL to my site at the bottom, and I have found out that we just don’t do that).</p>
<p>I have since removed the alteration, so all the LICENSE files are now correct, but the repos still don’t show the correct license.</p>
<p>This is an issue for me, as the license property of the repo is used by a third-party index (It’s been an issue with my repos for a long time, but this is the first time it’s caused me pain).</p>
<p>The question is simple: Is there some way to “reload” the repo, so the the fixed license is detected?</p>

**Comment by yamiacat** [comment]
<p>Hi <a class="mention" href="/chrismarshallny">@ChrisMarshallNY</a>! :wave: Welcome to the Community!</p>
<p>Did you replace the license by using the “Choose a license template” function?</p>
<p>(Step 4 in <a href="https://docs.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository#including-an-open-source-license-in-your-repository">this guide</a>.)</p>
<p>If so, that should have kicked off a rescan. I’m afraid it’s not possible for you to trigger a rescan manually any other way.</p>
<p>If you let me know the repositories affected, I can trigger a rescan of them from the back end!</p>

**Comment by ChrisMarshallNY** [comment]
<p>Thanks!</p>
<p>If you can do the rescan, that would be great!</p>
<p>I will look at the template for future work.</p>
<p>Here are the repos:</p>
<p>RiftValleySoftware/RVS_Generic_Swift_Toolbox<br>
RiftValleySoftware/RVS_Spinner<br>
RiftValleySoftware/RVS_ParseXMLDuration<br>
RiftValleySoftware/RVS_IPAddress<br>
RiftValleySoftware/RVS_BasicGCDTimer</p>
<p>Thanks!</p>

**Comment by yamiacat** [comment, marked as answer]
<p>I’ve queued a rescan of those repos, which could take up to 24 hours to run.</p>
<p>I’m not 100% sure it’s going to work as you’re missing the “Copyright © [year] [fullname]” line from your files:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://choosealicense.com/favicon.ico" class="site-icon" width="16" height="16">
      <a href="https://choosealicense.com/licenses/mit/" target="_blank" rel="noopener">Choose a License</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://choosealicense.com/licenses/mit/" target="_blank" rel="noopener">MIT License</a></h3>

<p>A short and simple permissive license with conditions only requiring preservation of copyright and license notices. Licensed works, modifications, and larger works may be distributed under different terms and without source code.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If it’s not showing as MIT after 24 hours, please add that line in and we’ll try again!</p>

**Comment by ChrisMarshallNY** [comment]
<p>Thanks!</p>
<p>I’ll add it to the files, nonetheless. I’ll let you know if it doesn’t take.</p>
<p><em><strong>UPDATE:</strong> Looks like that did the trick! Thanks!</em></p>
